src/app/Http/Requests/ChangePasswordRequest.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace Backpack\Base\app\Http\Requests;
+
+use Illuminate\Foundation\Http\FormRequest;
+use Illuminate\Support\Facades\Hash;
+
+class ChangePasswordRequest extends FormRequest
+{
+    /**
+     * Determine if the user is authorized to make this request.
+     *
+     * @return bool
+     */
+    public function authorize()
+    {
+        // only allow updates if the user is logged in
+        return \Auth::check();
+    }
+
+    /**
+     * Get the validation rules that apply to the request.
+     *
+     * @return array
+     */
+    public function rules()
+    {
+        return [
+            'old_password'         => 'required',
+            'new_password'         => 'required|min:6',
+            'confirm_password'     => 'required|same:new_password|min:6',
```

Comment:
Isn't that `min:6` somehwat redundant given that it *has* to conform to the new_password?